.gitignore

```diff
@@ -1,2 +1,3 @@
 /config/keys.js
-/node_modules
\ No newline at end of file
+/node_modules
+/frontend/node_modules
```

Comment:
the `/frontend/node_modules` line is technically superfluous because it's in the `frontend/.gitignore` but it being here is still good